### PR TITLE
Use the new Sensu Redis client & Transport API

### PR DESCRIPTION
**Pull request by portertech**
- performance improvements

- sentinel support

- Updated Sensu transport API, deferrable object

### CODE REVIEW

File: lib/sensu/server/sandbox.rb [hunk 1]
```diff
@@ -2,16 +2,16 @@ module Sensu
...
+      # `$SAFE` no longer supports levels > 1, so its use has been
+      # removed from this method. A single value is provided to the
+      # "sandbox".
       #
       # @param expression [String] to be evaluated.
       # @param value [Object] to provide the "sandbox" with.
       # @return [Object]
       def self.eval(expression, value=nil)
         result = Proc.new do
-          $SAFE = (RUBY_VERSION < "2.1.0" ? 4 : 3)
```
Comment (portertech) [inline lib/sensu/server/sandbox.rb]: Ruby 2.3.0 doesn't support $SAFE levels > 1, so let's just remove it. We will need to document eval filter attributes with a disclaimer, "don't do silly things".

File: sensu.gemspec [hunk 2]
```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |s|
   s.add_dependency "sensu-settings", "3.3.0"
   s.add_dependency "sensu-extension", "1.5.0"
   s.add_dependency "sensu-extensions", "1.4.0"
-  s.add_dependency "sensu-transport", "4.0.0"
+  s.add_dependency "sensu-transport", "5.0.0"
   s.add_dependency "sensu-spawn", "1.8.0"
-  s.add_dependency "em-redis-unified", "1.0.1"
```
Comment (portertech) [inline sensu.gemspec]: Happy to get off of this EM Redis fork.

Comment (agoddard) [inline sensu.gemspec]: \o/

**Comment by agoddard** [comment]
[oh yeah. this. will. do. just. fine. ](https://www.youtube.com/watch?v=cvPIwjX-pHo)

**Pull request merged**